src/api.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable max-len */
+/* eslint-disable prefer-promise-reject-errors */
```

Comment:
remove comment and fix lint. Please check your code and improve according this comment